### DISCUSSION TITLE
How-to delete a single test run without deleting the project?

### DISCUSSION THREAD

**Discussion by BrianBaggs** [test-results-reporter/reporter#248] [Q&A] [answered]
Hello!

I was able to delete a project using one of the discussion answers, but I was unable to figure out how to delete one single test run without deleting the entire project. I tried adding /runs/UUID after the /project/UUID but that did not work.

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @BrianBaggs , we don’t support it at the moment. But it is on our future roadmap.

**Reply by BrianBaggs** [reply]
Ok, thanks!
